src/twisted/web/http.py

```diff
@@ -2275,13 +2271,15 @@ def connectionMade(self):
         )
         self._networkProducer.registerProducer(self, True)
 
+    def dataReceived(self, data):
```

Comment:
thanks for the change.

I was expecting that this is already handled via the PolicyMixin ... byt it looks like it's not.

https://github.com/twisted/twisted/blob/10fc2f1cf0500f59497557d65b6751d473c40d85/src/twisted/protocols/policies.py#L620